model/version_activation.go

```diff
@@ -13,26 +13,82 @@ import (
 )
 
 func DoProjectActivation(ctx context.Context, id string, ts time.Time) (bool, error) {
```

Comment:
Overall this PR looks quite good! Thank you for it!

There's only a couple more things I think it needs to get merged.
1. We want to keep the default behavior of the repotracker only activating one version. To support activating multiple like this, a project setting should be added. [This PR](https://github.com/evergreen-ci/evergreen/pull/9356) is a good example. There is also a corresponding [UI PR](https://github.com/evergreen-ci/ui/pull/1090). I would recommend doing a separate PR for both and then after they are merged, consuming it in this PR.
2. The lookback doesn't have any bounds, it should probably have some type of bounds (maybe 10? I'm just spitballing a number but what is general an appropriate amount of repotracker versions skipped in your use-case?)

Reply:
> As for 2, I'm not sure I want any bounds. We do not want any version skipped at all.

That makes sense, but the bound isn't to necessarily have you skip a version. If your project right now only skips up to 2-3 versions, the bound is to prevent your project (or any that enable the setting) to not schedule an _outrageous_ /unexpected amount. For example, if someone rebases their branch (suddenly 500 commits) or suddenly pushes 50 commits to the branch.

If your workflow is pushing 50 commits, the bound could be a higher number. It's more so a protective caution.